solidity/contracts/bridge/Bridge.sol

```diff
@@ -854,7 +859,28 @@ contract Bridge is Ownable, EcdsaWalletOwner {
         );
     }
 
-    // TODO: updateMovingFundsParameters
+    /// @notice Updates parameters of moving funds.
+    /// @param movingFundsTxMaxTotalFee New value of the moving funds transaction
+    ///        max total fee in satoshis. It is the maximum amount of the total
+    ///        BTC transaction fee that is acceptable in a single moving funds
+    ///        transaction. This is a _total_ max fee for the entire moving
+    ///        funds transaction.
+    /// @param movingFundsTimeout New value of the moving funds timeout in
+    ///        seconds. It is the time after which the moving funds process can
+    ///        be reported as timed out. It is counted from the moment when the
+    ///        wallet was requested to move their funds and switched to the
+    ///        MovingFunds state.
+    /// @dev Requirements:
+    ///      - Moving funds timeout must be greater than zero
```

Comment:
Shouldn't we require that movingFundsTxMaxTotalFee > 0 ?

Reply:
Done in f52361d9df33814eeb9ed3ffa56562addb2c996b.